section3/exercises/arrays.js

```diff
@@ -96,21 +104,34 @@ YOU DO:
 Using the variables defined below, write a program that will tell a user if they
 will be able to call an Uber.
 
-The user can call an uber if they have 15% battery remaining, or more. In this case, it doesn't matter 
+The user can call an uber if they have 15% battery remaining, or more. In this case, it doesn't matter
 if the user has a charger at all, or what type.
 The can call an uber if they have a charger and it is a car charger.
 
 */
-var percentBatteryLeft = 12;
+var percentBatteryLeft = 100;
 var hasCharger = true;
 var chargerType = "car";
 
-// Write your conditional here
-
-
+// Conditional statement 'If' runs the log command when variables value meets the conditional operators evaluation
+if (percentBatteryLeft >= 15) {
+  console.log("Congrats! You can call an Uber!");
+// Second command executes if conditional operators in the second brackets evaluate to true
+} else if (percentBatteryLeft <= 14, hasCharger === true && chargerType === "car") {
+  console.log("Luckily, You can call an Uber!");
+//Third command executes if the first two conditions are not met and the statements in third bracket are true
+} else if (percentBatteryLeft <= 14, hasCharger === false && chargerType === "car") {
+  console.log("I'm sorry, You cannot call an Uber.");
+//Otherwise, if none of the previous conditions are met, the last statement will execute
+} else if (percentBatteryLeft <= 14, hasCharger === true && chargerType !== "car") {
+  console.log("I'm sorry, You cannot call an Uber.");
+//This statement runs if none of the variables meet any of the operators evaluations
+} else if (percentBatteryLeft <= 14, hasCharger === false && chargerType !== "car") {
+  console.log("I'm sorry, good luck finding a ride.");
+}
```

Comment:
These 3 `else if` statements are unnecessary. You can replace all of them with the below line of code:
```
} else {
  console.log("I'm sorry, good luck finding a ride.");
}
```

This is because you are using `else` as a "catch all" - if neither of your first 2 conditional statements are true on lines 117 and 120, then the code will always console log the "I'm sorry, good luck finding a ride." message.